examples/chart.md

```diff
@@ -2,12 +2,12 @@
 
 [last mile]: https://testingclouds.wordpress.com/2018/07/20/844/
 [stable chart]: https://github.com/helm/charts/tree/master/stable
-[Helm charts]: https://github.com/helm/charts
+[helm charts]: https://github.com/helm/charts
 [_minecraft_]: https://github.com/helm/charts/tree/master/stable/minecraft
 [plugin]: ../docs/plugins
 
 [Helm charts] aren't natively read by kustomize, but
-kustomize has a [plugin] system that allows one to
+kustomize has a built in HelmChartInflationGenerator that allows one to
```

Comment:
built-in is actually [a word](https://www.merriam-webster.com/dictionary/built-in).

we alternate between `builtin` and `built-in`, but avoid the two word form.